### PR TITLE
pktanon: update homepage

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

previous homepage, https://web.archive.org/web/20230402092616/http://www.tm.uka.de/software/pktanon/index.html
new one, http://www.tm.kit.edu/software/pktanon/download/index.html

### CODE REVIEW

**Comment by chenrui333** [comment]
There is another one `ps2eps` also needs some update, but I could not find the new artifact location (already sent out the author an email)

File: Formula/pktanon.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Pktanon < Formula
   desc "Packet trace anonymization"
-  homepage "https://www.tm.uka.de/software/pktanon/index.html"
-  url "https://www.tm.uka.de/software/pktanon/download/pktanon-1.4.0-dev.tar.gz"
+  homepage "http://www.tm.kit.edu/software/pktanon/index.html"
+  url "http://www.tm.kit.edu/software/pktanon/download/pktanon-1.4.0-dev.tar.gz"
```
Comment (Bo98) [inline Formula/pktanon.rb:4]: no HTTPS?

Comment (chenrui333) [reply Formula/pktanon.rb:4]: yeah, no https.

Comment (p-linnane) [reply Formula/pktanon.rb:4]: I just checked and HTTPS works for me on both links.

Comment (chenrui333) [reply Formula/pktanon.rb:4]: oh, interesting, it works for me now as well. 🤦

Comment (chenrui333) [reply Formula/pktanon.rb:4]: I should attach the curl test when i filed the PR 😄 
```
$ curl -I https://www.tm.kit.edu/software/pktanon/index.html
HTTP/1.1 200 OK
Date: Mon, 26 Jun 2023 14:48:45 GMT
Server: Apache
Accept-Ranges: bytes
Vary: Accept-Encoding
Content-Length: 5536
Content-Type: text/html
```

**Review by p-linnane — APPROVED** [review]

**Review by Bo98 — APPROVED** [review]

**Pull request merged**